backend/src/test/java/com/huellapositiva/util/TestData.java

```diff
@@ -268,6 +266,53 @@ public JpaProposal registerESALAndFinishedProposal() throws ParseException {
         return registerESALAndProposal(FINISHED);
     }
 
+    public JpaProposal registerESALAndProposalWithInscribedVolunteers() throws ParseException {
+        return registerESALAndProposalWithInscribedVolunteers(PUBLISHED);
+    }
+
+    @SneakyThrows
+    private JpaProposal registerESALAndProposalWithInscribedVolunteers(ProposalStatus proposalStatus) {
+
+        JpaVolunteer jpaVolunteer = createVolunteer(DEFAULT_EMAIL, DEFAULT_PASSWORD, Roles.VOLUNTEER);
+
+        JpaContactPerson contactPerson = createESALJpaContactPerson(DEFAULT_ESAL_CONTACT_PERSON_EMAIL, DEFAULT_PASSWORD);
+        JpaESAL esal = JpaESAL.builder().id(UUID.randomUUID().toString()).name(DEFAULT_ESAL).build();
+        createAndLinkESAL(contactPerson, esal);
+        JpaProposal jpaProposal = JpaProposal.builder()
+                .id(UUID.randomUUID().toString())
+                .title("Recogida de ropita")
+                .location(JpaLocation.builder()
+                        .id(UUID.randomUUID().toString())
+                        .province("Santa Cruz de Tenerife")
+                        .town("Santa Cruz de Tenerife")
+                        .address("Avenida Weyler 4").build())
+                .esal(esal)
+                .startingProposalDate(new SimpleDateFormat("dd-MM-yyyy").parse("20-08-2020"))
```

Comment:
You can create a SimpleDateFormat object and reuse it inside the function. You cannot reuse at class level since it is not thread safe

Reply:
SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd-MM-yyyy");

Created inside the method. And reused it there.